grantnav/frontend/templates/funder.html

```diff
@@ -47,7 +47,7 @@ <h1 class="funder-header text-center">{{ funder.name }}</h1>
       </div>
     </div>
     <div class='spacer-4'>
-      <div class="box box--teal">
```

Comment:
I don't see a request in the issue to change the colour to black instead of teal?

Reply:
@michaelwood it is in the change that @drkane has done in the design system.

Reply:
Sorry - that was me changing how the templates are displayed in the design system. The box should stay teal.

Reply:
Thanks @drkane. I have now deleted the commit with these changes. I have checked locally and it is teal and underlined.